packages/esm-active-visits-app/src/home-page-tiles/total-visits-metric-tile/total-visits-tile.resources.tsx

```diff
@@ -0,0 +1,20 @@
+import { openmrsFetch, restBaseUrl } from '@openmrs/esm-framework';
+import useSWR from 'swr';
+import dayjs from 'dayjs';
+import { type Visit } from '../../types/index';
+
+const useTotalVisits = () => {
+  const omrsDateFormat = 'YYYY-MM-DDTHH:mm:ss.SSSZZ';
+  const currentVisitDate = dayjs(new Date().setHours(0, 0, 0, 0)).format(omrsDateFormat);
+  const customRepresentation = 'custom:(uuid,startDatetime,stopDatetime)';
+
+  const visitsUrl = `${restBaseUrl}/visit?includeInactive=true&v=${customRepresentation}&fromStartDate=${dayjs(
+    currentVisitDate,
+  ).format('YYYY-MM-DD')}`;
```

Comment:
Why are we going through all these gymnastics with `currentVisitDate` when `dayjs().format('YYYY-MM-DD')` would do the same thing?